### PR TITLE
Add impossible page

### PR DESCRIPTION
**Pull request by jcnicolai**
Please review this, @casey-collab

### CODE REVIEW

**Comment by casey-collab** [comment]
Unfortunately this pull request could not be automatically reviewed due to technical difficulty.

**Comment by jcnicolai** [comment]
@casey-collab

File: impossible.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+IM POS SI BLE
```
Comment (casey-collab) [inline impossible.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: impossible.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  - not able to occur, exist, or be done
+  - improbable, utterly impractical
+  - unable to be, exist, happen
+  - not capable of being accomplished
+
+You begin to [ponder](../ponder-about-life.md).
```
Comment (casey-collab) [inline impossible.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: impossible.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Suddenly you have a flashback to being in a [forest](../forest/forest.md)
```
Comment (casey-collab) [inline impossible.md]: This link is also invalid.

File: impossible.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+Have I become a [front-end web developer](https://www.udacity.com/course/front-end-web-developer-nanodegree--nd001)
+
+Is that a [Python](https://en.wikipedia.org/wiki/Pythonidae) on my desktop?
+
+No. Wait. Am I a [full stack web developer](https://www.udacity.com/course/full-stack-web-developer-nanodegree--nd004)?
+
+This is not possible. I need to look more closely at the data in front of me.
+Surely I have not become a [data analyst](https://www.udacity.com/course/data-analyst-nanodegree--nd002).
+
+Is this merely a [dream within a dream](../dream-in-dream/inception.md)
```
Comment (casey-collab) [inline impossible.md]: This link is also invalid.

File: impossible.md (hunk 1 above)
Comment (casey-collab) [inline impossible.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by jcnicolai** [comment]
@casey-collab

File: impossible.md (hunk 1 above)
Comment (casey-collab) [inline impossible.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: impossible.md (hunk 2 above)
Comment (casey-collab) [inline impossible.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: impossible.md (hunk 3 above)
Comment (casey-collab) [inline impossible.md]: This link is also invalid.

File: impossible.md (hunk 4 above)
Comment (casey-collab) [inline impossible.md]: This link is also invalid.

File: impossible.md (hunk 1 above)
Comment (casey-collab) [inline impossible.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by jcnicolai** [comment]
Hello. I feel stuck. I'm not sure where to go from here. I think I have addressed the bugs/issues. But am not finding a way to continue with the merge.

Can you provide some feedback or guidance to help me?

Thanks,
JC

**Comment by jcnicolai** [comment]
Going to start over and try this lesson from scratch.

**Pull request closed without merging**